static/app/views/issueDetails/groupEvents.tsx

```diff
@@ -24,7 +24,14 @@ interface State {
   query: string;
 }
 
-const excludedTags = ['environment', 'issue', 'issue.id', 'performance.issue_ids'];
+const excludedTags = [
+  'environment',
+  'issue',
+  'issue.id',
+  'performance.issue_ids',
+  'transaction.op',
+  'transaction.status',
+];
```

Comment:
occurrences don't have these transaction-specific properties, so I'm removing them from the search in the all events tab